vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -37,7 +37,7 @@
 
 | Term |  Description
 |:--|:--
-| application composition platform (ACP) | Any development platform that supports the cataloging and management of composable and packaged components, where new components can be added through custom development or imported from existing assets; it governs the life cycles of both the components and the applications built from their modular assembly and deployment |
+| application composition platform (ACP) | Any development platform that supports the cataloging and management of composable and packaged components, where new components can be added through custom development or imported from existing assets. It governs the life cycles of both the components and the applications built from their modular assembly and deployment. |
```

Comment:
I know this is based on the Gartner papers, but don't think this explains what Entando can do very well

Reply:
this is the definition of an acp, not an entando product